patches/components-page_info-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
+index 8fecfec436e938370d5a52baa54fe1b312bbf832..2ecc7b8e8356d76b1f08ee5e1d37ce9b929b5100 100644
+--- a/components/page_info/BUILD.gn
++++ b/components/page_info/BUILD.gn
+@@ -37,6 +37,7 @@ static_library("page_info") {
+     "//services/device/public/cpp:device_features",
+     "//services/metrics/public/cpp:ukm_builders",
+   ]
++  deps += [ "//brave/components/vector_icons" ]
```

Comment:
this should be //brave/components/page_info to be more extensible

Reply:
Addressed by cbb4586810c1f95018f8f6f34359aedd89d9f1c1